atox/src/main/kotlin/PendingIntentCompat.kt

```diff
@@ -20,7 +20,13 @@ object PendingIntentCompat {
         if (Build.VERSION.SDK_INT < Build.VERSION_CODES.M) {
             PendingIntent.getBroadcast(context, requestCode, intent, flags)
         } else {
-            val mutabilityFlag = if (mutable) PendingIntent.FLAG_MUTABLE else PendingIntent.FLAG_IMMUTABLE
+            val mutabilityFlag =
+                if (mutable) {
+                    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.S) PendingIntent.FLAG_MUTABLE
```

Comment:
Can you put this and the next line on the same row? Either that or add braces. :P
Only single-line if constructions may leave out the braces. I haven't found a style rule to set this up, but it's how the rest of the codebase is written. :x